### ISSUE TITLE
Tool doesn't work anymore.

### ISSUE THREAD

**Issue by RandomUser779** [RORIdev/moesharp#2]
Hello, thanks for your hard work, but recently I tried executing the program but I got an error which roughly states that it only supports Chrome version 78. If you are still willing to manage this, is there a solution to this? A picture of the error is attached.
![image](https://user-images.githubusercontent.com/72160539/94723738-caa3c500-038b-11eb-9779-aa261d537ffe.png)

**Comment by alikindsys** [comment]
phew. haven't messed with moesharp in ages. I may try to fix it and make it better but that'd require quite some time.
Currently even if chromedriver were updated the tool wouldn't work, since anime twist changed how user validation works since a while ago.
thanks for reminding this even existed.

**Comment by RandomUser779** [comment]
Thanks for attending to this relatively old project! Glad to hear that a fix might come.

**Comment by alikindsys** [comment]
Little status update, i've begun on the rewrite of it as of a couple of hours ago.
This update has support for both chrome and firefox, and you can choose once you start it for the first time.
I won't provide chromedriver anymore to minimize the chances of that happening again.
Its now written in kotlin so the only dependency you'll need is java 8. I'll cut up on the setup process as much as i can.
This is a sneak peak of it : 
![](https://i.imgur.com/P1Cg7eN.png)

**Comment by RandomUser779** [comment]
Alright, cool. So I will just have to download java 8 from links like this https://www.java.com/en/download/manual.jsp right?

**Comment by alikindsys** [comment]
yea it should be it.

**Comment by alikindsys** [comment]
Good news. I've managed to download files, and the tool is almost ready for release. Just need some more testing!
* Partial downloads are now a thing. moekt will resume downloads if you run it again (in case of an error happening and the program closing.)

```
Getting episodes for : [Eromanga-sensei]
Current episode : 1
Current episode : 2
Current episode : 3
Current episode : 4
Current episode : 5
Current episode : 6
Current episode : 7
Current episode : 8
Current episode : 9
Current episode : 10
Current episode : 11
Current episode : 12
Finished getting episodes for Eromanga-sensei. Count : 12
[1/12] Downloading Eromanga-sensei - Episode 1 to [C:\Users\roridev\Development\moesharp\anime\Eromanga-sensei\1.mp4]
Resuming download of [Episode 1] 0/340838867
Status Code = 206
```

**Comment by alikindsys** [comment]
Testing is completed. The tool is confirmed to work. Setup process will happen once you start it. Will be packaged and made public shortly after cleanup process is complete. With that i'm glad to announce that this issue, is closed.

**Comment by alikindsys** [comment]
PSA : It has been released. 
[Download Link](https://github.com/RORIdev/moesharp/releases/tag/kt-1.0)

**Comment by RandomUser779** [comment]
Cool, that's fast! Thank you for your hard work.

**Comment by alikindsys** [comment]
> Cool, that's fast! Thank you for your hard work.

No problem! Let me know if any problems arise.

**Comment by alikindsys** [comment]
just letting people know that i released an update with some optimizations that would make the tool *faster*. you just need to change the `moekt.jar` file for the new one, and re-run the command. config file is untouched and will work without problems.

**Comment by RandomUser779** [comment]
I tried the program but I have another problem. The episode corresponding to the link I input was captured, but the program seems to hit an error and stopped loading. I tried both versions of moekt.jar files, but the issues were the same. Here is a screenshot of it, what might be the problem?
![Problem](https://user-images.githubusercontent.com/72160539/94945294-0c0fae00-050d-11eb-9c5f-00bf0ad38907.png)

**Comment by RandomUser779** [comment]
I tried the program but I have another problem. The episode corresponding to the link I input was captured, but the program seems to hit an error and stopped loading. I tried both versions of moekt.jar files, but the issues were the same. Here is a screenshot of it, what might be the problem?
![Problem](https://user-images.githubusercontent.com/72160539/94946182-52b1d800-050e-11eb-84ac-c807e36fde32.png)
